### ISSUE TITLE
Search needs a label

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#13095]
The search currently does not have a label. It should, for accessibility reasons.

```html
<input id="filter-menus" class="form-control" type="search" placeholder="Search">
```

While "search" is referenced several times, the `<input>` still needs a `<label>` or an `aria-label="search"` (which would work well in this case).

For details, see https://dequeuniversity.com/rules/axe/3.4/label
